src/components/ListItem/ListItem.component.tsx

```
@@ -0,0 +1,73 @@
+    isButton,
+    status,
+    endContent,
+    sx,
+}: ListProps) => {
+    const { pathname } = useLocation();
+    const isActiveRoute = React.useMemo(() => {
+        const routePath = `/${text.toLowerCase()}`;
+
+        return (
+            pathname === routePath ||
+            pathname.startsWith(routePath + '/') ||
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)